examples/android_instrumentation_test/WORKSPACE

```diff
@@ -3,7 +3,7 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 android_sdk_repository(
     name = "androidsdk",
     api_level = 28,
-    build_tools_version = "28.0.2",
+    build_tools_version = "30.0.3",
```

Comment:
We can remove this version pinning altogether.

Reply:
Updated